functions.Rmd

```diff
@@ -14,6 +14,24 @@ row_adder()
 permutation()
 ```
 
+## Anonymous functions
+
+If using an anonymous or inline function (i.e. a function that hasn't been given a name by assigning it to a variable), prefer the new lambda syntax: `\(x) x + 1`.
+
+```R
+# Good
+map(xs, \(x) mean((x + 5)^2))
+
+# Bad
+map(xs, function(x) mean((x + 5)^2))
```

Comment:
Grrrrr I might fight you on this and say that either `\(x)` or `function(x)` are ok here

I tend to find `function(x)` more readable in many cases, _particularly_ if it spans multiple lines like:

```r
map(xs, function(x, y) {
  mean((x + y)^2)
})
```

But phasing out `~` does seem reasonable